proto/v1/sbom.proto

```diff
@@ -0,0 +1,389 @@
+    SOURCE = 1;
+    // The file is a compiled object, target image or binary executable (.o, .a,
+    // etc.)
+    BINARY = 2;
+    // The file represents an archive (.tar, .jar, etc.)
+    ARCHIVE = 3;
+    // The file is associated with a specific application type (MIME type of
+    // application/*)
+    APPLICATION = 4;
+    // The file is associated with an audio file (MIME type of audio/* , e.g.
+    // .mp3)
+    AUDIO = 5;
+    // The file is associated with an picture image file (MIME type of image/*,
+    // e.g., .jpg, .gif)
+    IMAGE = 6;
+    // The file is human readable text file (MIME type of text/*)
+    TEXT = 7;
+    // The file is associated with a video file type (MIME type of video/*)
+    VIDEO = 8;
+    // The file serves as documentation
+    DOCUMENTATION = 9;
+    // The file is an SPDX document
+    SPDX = 10;
+    // The file doesn't fit into the above categories (generated artifacts, data
+    // files, etc.)
+    OTHER = 11;
+  }
+  // This field provides information about the type of file identified
+  FileType file_type = 2;
+  // Provide a unique identifier to match analysis information on each specific
+  // file in a package
+  repeated string checksum = 3;
+}
+
+// FileOccurrence represents an SPDX File Information
+// section: https://spdx.github.io/spdx-spec/4-file-information/
+message FileOccurrence {
+  // Uniquely identify any element in an SPDX document which may be referenced
+  // by other elements
+  string id = 1;
+
+  // This field contains the license the SPDX file creator has concluded as
+  // governing the file or alternative values if the governing license cannot be
+  // determined
+  string license_concluded = 2;
+  // This field contains the license information actually found in the file, if
+  // any
+  repeated string files_license_info = 3;
+  // This field provides a place for the SPDX file creator to record any
+  // relevant background references or analysis that went in to arriving at the
+  // Concluded License for a file
+  string license_comments = 4;
+  // Identify the copyright holder of the file, as well as any dates present
+  string copyright = 5;
+  // This field provides a place for the SPDX file creator to record any general
+  // comments about the file
+  string comment = 6;
+  // This field provides a place for the SPDX file creator to record license
+  // notices or other such related notices found in the file
+  string notice = 7;
+  // This field provides a place for the SPDX file creator to record file
+  // contributors
+  repeated string contributors = 8;
+  // This field provides a place for the SPDX data creator to record, at the
+  // file level, acknowledgements that may be needed to be communicated in
+  // some contexts
+  repeated string attributions = 9;
+}
+
+// RelationshipNote represents an SPDX Relationship
+// section:
+// https://spdx.github.io/spdx-spec/7-relationships-between-SPDX-elements/
+message RelationshipNote {}
```

Comment:
this is something we postponed the discussion on for the best way to represent, let's remove this for now and re-open the conversation internally? ditto for RelationshipOccurrence